homeworks/artem.bilenko_brilux/homework_3/homework_3_task_1.js

```diff
@@ -0,0 +1,43 @@
+const readline = require('readline');
+
+const inputData = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+inputData.question('Enter the number to square it: ', (x) => {
+    inputData.question('Enter the max value of array to create it: ', (maxValue) => {
+        inputData.question('Enter the step of array to create it: ', (step) => {
+            inputData.close();
+
+            const pow = () => x * x;
```

Comment:
function supposed to be standalone unit, not producing any side effects, in your case it should be smth like `(arg) => arg * arg`